src/components/code-block/CodeBlock.js

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";
+import { dracula } from "react-syntax-highlighter/dist/esm/styles/prism";
+import "./CodeBlock.css";
+
+const context = require.context("!raw-loader!../../sdks", true);
```

Comment:
I'm not sure, but maybe we can even move it to a global file so it will read only once (although since Docusaurus is compiled to be static, there's no real reason for that)

Reply:
We could do it, but I don't think it gives much performance improvement at all, and it's only used for the code block - I don't think we will be generating a tree of files anywhere - so for this reason I think it's better to keep it in the file. 

If we ever extend the functionality of this, then I'd say we move it into the utils. It's little work.